PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -151,7 +153,7 @@ extension ArchivedItemsListViewModel {
     func filterButton(with id: ItemsListFilter) -> TopicChipPresenter {
```

Comment:
Not directly affected by this pull request, but what's your opinion on having the argument label `id` vs updating it to something like `filter`?

Reply:
def like `filter` better - will update!